extractor/src/main/java/org/schabi/newpipe/extractor/utils/StringUtils.java

```diff
@@ -0,0 +1,48 @@
+package org.schabi.newpipe.extractor.utils;
+
+
+import javax.annotation.Nonnull;
+
+public class StringUtils {
+
+    private StringUtils() {
+    }
+
+    /**
+     * @param string The string to search in
+     * @param start A string from which to start searching.
+     * @return A substring where each '{' matches a '}'
+     * @throws IndexOutOfBoundsException If {@code string} does not contain {@code start}
+     */
+    @Nonnull
+    public static String matchToClosingParenthesis(@Nonnull final String string, @Nonnull final String start) {
```

Comment:
Mmmh, maybe add a couple of simple tests for this function

Reply:
done